### DISCUSSION TITLE
[x][closed][fairtracks] Daniel: What is the difference between biomaterial_type and sample_type? [Sample]

### DISCUSSION THREAD

**Discussion by sveinugu** [fairtracks/fairtracks_standard#90] [Team Posts]
.

**Comment by sveinugu** [comment]
@dzerbino This is directly from IHEC: https://github.com/IHEC/ihec-metadata/blob/master/specs/Ihec_metadata_specification.md#samples, and is a controlled vocabulary of:

* Cell line
* Primary cell
* Primary cell culture
* Primary tissue

The choice of biomaterial_type has direct impacts on what to use in sample_type. "Primary tissue" would lead to e.g. "Brain" as sample_type, while "Cell line" would lead to e.g. "H1-hESC". It could perhaps be called sample category, or similar.

**Comment by sveinugu** [comment]
@dzerbino One issue here is that we would need to expand the vocabulary for e.g. tumor samples, and I don't feel like an authority on what nomenclature to use.

**Comment by dzerbino** [comment]
OK, this should be made explicit in the documentation. We should have a look at ICGC sample descriptions.

**Comment by sveinugu** [comment]
Created issue: https://github.com/fairtracks/fairtracks_standard/issues/15. Closing
